<!DOCTYPE html>
<html lang="en-US">
<head>

    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="theme" content="hugo-academic-group">

    
    

    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <script src="http://rapeseed.zju.edu.cn//js/hugo-academic-group.js"></script>

    <link rel="stylesheet" href="http://rapeseed.zju.edu.cn//css/bootstrap.min.css">
    <script src="http://rapeseed.zju.edu.cn//js/bootstrap.min.js"></script>  

    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.6.0/styles/default.min.css">

    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/github-fork-ribbon-css/0.2.2/gh-fork-ribbon.min.css" />
    
    
    <script src="http://rapeseed.zju.edu.cn//js/highlight.pack.js"></script>
    <script>hljs.initHighlightingOnLoad();</script>


    <link rel="stylesheet" href="http://rapeseed.zju.edu.cn//css/font-awesome.min.css">
    <link rel="stylesheet" href="http://rapeseed.zju.edu.cn//css/academicons.min.css">
    <link rel="stylesheet" href="http://rapeseed.zju.edu.cn//css/biaslab-font.css">
    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:100,300,400,700|Merriweather:100,400,700|Roboto+Mono">
    <link rel="stylesheet" href="http://rapeseed.zju.edu.cn//css/hugo-academic-group.css">

    


    <link rel="shortcut icon" href="http://rapeseed.zju.edu.cn//img/favicon.ico" type="image/x-icon">
    <link rel="canonical" href="http://rapeseed.zju.edu.cn/cloud/gwas/">

    <title> | JiangLX Lab</title>

</head>

<body>
<div class="home-anchor" id="home"></div>


<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
    <div class="container">

        
        <div class="navbar-header">
            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".navbar-collapse" aria-expanded="false">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>

            <div class="navbar-brand">
                
                    <a class="logo" href="http://www.zju.edu.cn/">
                        <img src="http://rapeseed.zju.edu.cn/img/zjulogo.svg" alt="University logo"></img>
                    </a>
                
                
                 
                    
                    
                        <a href="http://rapeseed.zju.edu.cn/">Li-Xi Jiang Lab</a>
                    
    
                
            </div>
        </div>

        
        <div class="collapse navbar-collapse" id="#navbar-collapse-1">

            
            <ul class="nav navbar-nav navbar-right">
                
                    
                        <li class="nav-item"><a data-scroll href="http://rapeseed.zju.edu.cn/#home">Home</a></li>
                    
                
                    
                        <li class="nav-item"><a data-scroll href="http://rapeseed.zju.edu.cn/#projects">Projects</a></li>
                    
                
                    
                        <li class="nav-item"><a data-scroll href="http://rapeseed.zju.edu.cn/#publications">Publications</a></li>
                    
                
                        
                        
                        <li class="dropdown">
                            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Cloud <span class="caret"></span></a>
                            <ul class="dropdown-menu">
                              
                                <li><a href="http://rapeseed.zju.edu.cn/cloud/bnasnpdb">BnaSNPDB</a></li>
                              
                                <li><a href="http://rapeseed.zju.edu.cn/cloud/gwas">Bna_GWAS</a></li>
                              
                            </ul>
                          </li>
                    
                
                    
                        <li class="nav-item"><a data-scroll href="http://rapeseed.zju.edu.cn/#members">Members</a></li>
                    
                
                    
                        <li class="nav-item"><a data-scroll href="http://rapeseed.zju.edu.cn/class/class">Teaching</a></li>
                    
                
                    
                        <li class="nav-item"><a data-scroll href="http://rapeseed.zju.edu.cn/#posts">Blog</a></li>
                    
                
                    
                        <li class="nav-item"><a data-scroll href="http://rapeseed.zju.edu.cn/#contact">Contact</a></li>
                    
                
                
            </ul>

        </div>
    </div>
</nav>


<div class="container">
    <h1 id="font-facetime-colorgreen-size6introductionfont"><!-- raw HTML omitted -->Introduction<!-- raw HTML omitted --></h1>
<p><!-- raw HTML omitted -->Rapeseed (<em><strong>Brassica napus</strong></em> L.) is an important source of edible oil and protein-rich livestock feed in the world. <em><strong>B. napus</strong></em> (AACC) was ancestrally originated from an interspecific hybridization between two diploid progenitors, <em><strong>B. rapa</strong></em> (AA) (n = 10) and <em><strong>B. oleracea</strong></em> (CC) (n = 9), less than 7500 years ago. In our previous study, we resequenced a world-wide collection of <code>991</code> <em><strong>B. napus</strong></em> gerplasm accessions, including 658 winter types, 145 semi-winter types and 188 spring types, from 39 countries (<a href="https://www.sciencedirect.com/science/article/pii/S1674205218303435?via%3Dihub"><strong>Wu et al., 2019</strong></a>).</p>
<figure>
    <img src="http://rapeseed.zju.edu.cn/img/Bna_map.png" width="1000"/> 
</figure>

<p>In genetics, a genome-wide association study (<strong>GWAS</strong>), also known as whole genome association study (WGAS), is an observational study of a genome-wide set of genetic variants in different individuals to see if any variant is associated with a trait. <strong>GWAS</strong> typically focus on associations between single-nucleotide polymorphisms (<strong>SNPs</strong>) and traits like major agronomic traits.</p>
<p>In order to make better use of this huge <em><strong>B. napus</strong></em> gerplasm accessions, we develop this interactive application (<!-- raw HTML omitted --><strong>Bna_GWAS_Cloud</strong><!-- raw HTML omitted -->), you can also access it from http://121.41.229.126:3838/gwas/, in <a href="https://www.r-project.org/"><strong>R</strong></a> with <a href="https://shiny.rstudio.com/"><strong><code>Shiny</code></strong></a>. This aaplication can conduct <strong>GWAS</strong>, visualization of <strong>GWAS</strong> results (Manhattan plot and QQ plot), extraction of significant genes and annotation of genes<!-- raw HTML omitted -->.</p>
<h1 id="font-facetime-colorgreen-size6data-inputfont"><!-- raw HTML omitted -->Data input<!-- raw HTML omitted --></h1>
<h2 id="font-colorgreen-size5-facetimephenotype-data-txtfont"><!-- raw HTML omitted -->phenotype data (.txt)<!-- raw HTML omitted --></h2>
<blockquote>
<p><strong><!-- raw HTML omitted --> Noted: Your Samples Uploaded MUST Be The 300 Core Collection Samples Used Here ! So If Some Samples Are Not In Your LIST, You Need Add Them In Your List, And Set The Value NA. If Some Samples In Your List Are Not In The 300 Core Collection Samples Here, JUST REMOVE THEM!<!-- raw HTML omitted --></strong></p>
</blockquote>
<blockquote>
<p><!-- raw HTML omitted -->I highly recommended first download the example of the expected input phenotype dataset below, and then replace the phenotype values with you own data<!-- raw HTML omitted -->.</p>
</blockquote>
<p><!-- raw HTML omitted -->You just need upload your phenotype data to run <strong>GWAS</strong>. Here we just use the <strong><code>300</code></strong> core collection gerplasm which represent the most of genetic resources of 1000 <em><strong>B. napus</strong></em> gerplasm accessions. an example of the expected input data format is present as below:</p>
<table>
<thead>
<tr>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td>R4157</td>
<td>0.859791123</td>
</tr>
<tr>
<td>R4158</td>
<td>0.87369142</td>
</tr>
<tr>
<td>R4163</td>
<td>0.842593709</td>
</tr>
<tr>
<td>R4168</td>
<td>0.884782609</td>
</tr>
<tr>
<td>R4171</td>
<td>NA</td>
</tr>
<tr>
<td>R4176</td>
<td>0.885619807</td>
</tr>
<tr>
<td>R4177</td>
<td>0.885884455</td>
</tr>
<tr>
<td>R4179</td>
<td>0.879374612</td>
</tr>
<tr>
<td>R4180</td>
<td>0.878567797</td>
</tr>
<tr>
<td>R4182</td>
<td>0.868825911</td>
</tr>
<tr>
<td>&hellip;</td>
<td>&hellip;</td>
</tr>
</tbody>
</table>
<p>Where, column one correspond to samples, column two correspond to phenotype values.</p>
<p>An example of the expected input phenotype dataset can be accessible <a href="https://raw.githubusercontent.com/YTLogos/pic_link/master/sample_phenotype.txt"><strong>here</strong></a>.<!-- raw HTML omitted --></p>
<h2 id="font-colorgreen-szie5-facetimeother-parametersfont"><!-- raw HTML omitted -->Other parameters<!-- raw HTML omitted --></h2>
<p><!-- raw HTML omitted -->Next you need enter your trait name (<code>recommended</code>) (default: Bna_trait). Now just support the <a href="https://genome.sph.umich.edu/wiki/EMMAX"><strong>EMMAX</strong></a> model. After all the prepared works are ready, then clink <strong>Run Analysis</strong> to start GWAS.<!-- raw HTML omitted --></p>
<h1 id="font-colorgreen-size6-facetimevisualizationfont"><!-- raw HTML omitted -->Visualization<!-- raw HTML omitted --></h1>
<p><!-- raw HTML omitted -->For the <code>Visualization</code> section in this App, it is aiming to visualize the Manhattan plot and QQ plot. You can choose the alternate colors for alternate chromosomes and p-value threshold (default: p-value=5).<!-- raw HTML omitted --></p>
<h1 id="font-colorgreen-size6-facetimeextractionfont"><!-- raw HTML omitted -->Extraction<!-- raw HTML omitted --></h1>
<p><!-- raw HTML omitted -->The extraction of significant genes is based on the significant p-value of SNPs. So here you need choose the p-value threshold and the distance up/down-stream of SNPs (<code>recommended</code>) (default: 75kb).<!-- raw HTML omitted --></p>
<h1 id="font-colorgreen-size6-facetimeannotationfont"><!-- raw HTML omitted -->Annotation<!-- raw HTML omitted --></h1>
<p><!-- raw HTML omitted -->This section is designed for gene annotation based on different databases (<code>eggNOG</code>, <code>GO</code>, <code>KEGG</code>, <code>NR</code>, etc.).<!-- raw HTML omitted --></p>
<p>You can click <a href="http://121.41.229.126:3838/gwas">Here</a> and <a href="http://rapeseed.zju.edu.cn:3838/gwas">Here</a>  to access the <strong>Bna-GWAS-Cloud</strong>.</p>

</div>

<footer class="site-footer">
    <div class="container">
        <p class="powered-by">

            © JiangLX Lab, 2020 &middot; 

            Partially powered by the <a href="https://github.com/gcushen/hugo-academic" target="_blank">Academic theme</a> for <a href="http://gohugo.io" target="_blank">Hugo</a>.

            <span class="pull-right"><a href="#home" id="back_to_top"><span class="button_icon"><i class="fa fa-chevron-up fa-2x" aria-hidden="true"></i></span></a></span>

        </p>
    </div>
</footer>

<script src="//cdnjs.cloudflare.com/ajax/libs/gsap/1.18.4/TweenMax.min.js"></script>
<script src="//cdnjs.cloudflare.com/ajax/libs/gsap/latest/plugins/ScrollToPlugin.min.js"></script>




<script type="text/x-mathjax-config">
 MathJax.Hub.Config({ tex2jax: { inlineMath: [['$','$'], ['\\(','\\)']] } });
</script>

<script type="text/javascript" async
        src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS_CHTML">
</script>

<script type="text/javascript" src="//rf.revolvermaps.com/0/0/8.js?i=5l6fbkx4wz1&amp;m=0&amp;c=ff0000&amp;cr1=ffffff&amp;f=arial&amp;l=33" async="async"></script>

</body>
</html>

